test/mip_test.c

```diff
@@ -3,6 +3,8 @@
 #define MG_ENABLE_TCPIP 1
 #define MG_ENABLE_PACKED_FS 0
 
+#define MIP_TEST
```

Comment:
This will disable normal behavior, if it is for some test only please comment, I think it will not test what is later used in real life